### PR TITLE
Update Android SDK Build Tools version

### PR DESCRIPTION
**Pull request by lucasbento**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 25
```
Comment (Minishlink) [inline android/build.gradle]: Same thing :)
- 25.0.2 is the lastest
- compileSdkVersion should stay at 23, react-native-linear-gradient doesn't take advantage of features that needs API 24 or 25

**Comment by dgladkov** [comment]
React Native template project still uses those SDK/build tools versions:
https://github.com/facebook/react-native/blob/12de9729bd7217bac225b274b8475d8c36334cb8/local-cli/templates/HelloWorld/android/app/build.gradle#L86-L87

And they are mentioned in the official installation tutorial:
https://facebook.github.io/react-native/docs/getting-started.html

While technically using newer SDK target and build tools is possible, I believe this project should remain consistent with React Native and it's ecosystem.

Closing this PR, please feel free to reopen it when React Native decides to bump the versions:)

**Comment by Shion093** [comment]
Hi guys, build still fails, need to open up android folder and do the change manually, please update build tools on master. This also affects implementation on CI, because we can't change versions on CI server.

Thanks for your help and this awesome lib

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugApk'.
   > A problem occurred configuring project ':react-native-linear-gradient'.
      > The SDK Build Tools revision (23.0.1) is too low for project ':react-native-linear-gradient'. Minimum required is 25.0.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

**Comment by Minishlink** [comment]
@Shion093 If you want to override the build tools of your dependencies, there's a trick ;) I've just written a blog post on this : https://blog.bam.tech/developper-news/boost-compilation-time-react-native-android-app-latest-android-build-tools

**Pull request closed without merging**